go/client/cmd_login.go

```diff
@@ -79,7 +79,7 @@ func (c *CmdLogin) Run() error {
 
 	err = client.Login(ctx,
 		keybase1.LoginArg{
```

Comment:
I still feel that an empty string should just be an empty string and not use a pointer to a string with the pointer nil, but if this makes the JS significantly cleaner, then II'm ok.